src/HealthBase.jl

```diff
@@ -12,6 +12,10 @@ function __init__()
             if isnothing(get_extension(HealthBase, :HealthBaseDrWatsonExt))
                 _extension_message("DrWatson", cohortsdir, io)
             end
+        elseif exc.f == HealthTable
+            if isnothing(get_extension(HealthBase, :HealthBaseOMOPCDMExt))
```

Comment:
Can you check this error message? I think it'll give you a bad helper message right now with the "and" statement. Might be best to say "OMOPCommonDataModel, DataFrames".